base_rest/components/service.py

```diff
@@ -93,6 +93,7 @@ def _log_call(self, func, params, secure_params, res):
 
     def _get_validator(self, validator_method):
         if not hasattr(self, validator_method):
+            _logger.error("Missing validator %s", validator_method)
```

Comment:
@dreispt Validators are only required for input messages. An error is raised or log.warning is called into the _secure_input and _secure_output methods. IMO this change is useless.

Reply:
This might not be the best fix, but a missing output validator does not log a warning, it raises a traceback that does not give specific information on what expected validator is missing, sending the develop down into a debug rabbit hole.

Reply:
> a missing output validator does not log a warning, it raises a traceback

@dreispt That is surprising. Can you provide more details? There is code to log a warning in that case:

https://github.com/OCA/rest-framework/blob/28a8436dea7fbf69b5796948ccdc5d2df56e5c0c/base_rest/components/service.py#L161-L169

Reply:
IIRC that is true only for missing input validators, not for output validator. But I'll double check.